hypothesis/via
# Add wildcard matching for domain blocklist

We want to be able to block sites like: `*.terrible.com`, but currently we can only do exact matches.

### Testing notes

 * Run `make dev`
 * Observe that `*.example.net` is in `via/default-blocklist.txt`
 * Goto: http://localhost:9080/anything.example.net
 * You should see a blocked page
 * Other pages should work like normal
